script.js

```diff
@@ -1,12 +1,236 @@
 //You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+async function  setup() {
+  // const allEpisodes = getAllEpisodes();
+  const rootElem = document.getElementById("root");
+  const navBar = document.getElementById("navbar")
+  const input = document.createElement("input")
+  navBar.appendChild(input)
+  const pCount = document.createElement("p")
+  navBar.appendChild(pCount)
+  const select = document.getElementById("select")
+
+
+  function titleCodeGenerator(episode){
+    let seasonCode = (episode.season < 10) ? '0' + episode.season : episode.season;
+    let episodeCode = (episode.number < 10) ? '0' + episode.number : episode.number;
+    return `S${seasonCode}E${episodeCode}`;
+  }
+
+
+  function addEpisodes(episode) {
+
+    let divCard = document.createElement('div')
+    divCard.classList.add('cardEp')
+    let div = document.createElement('div')
+    div.classList.add('cardTitleEp')
+
+    episode.season ? div.innerText = `${episode.name} - ${titleCodeGenerator(episode)}` : div.innerText = `${episode.name}`
+
+    let imgCard = document.createElement('img')
+    imgCard.classList.add("cardImgEp")
+
+    imgCard.src=`${episode.image !== null ? episode.image.medium : "https://www.facultatieve-technologies.com/wp-content/uploads/No-image-200px.png"}`
+
+    let divCardSummery = document.createElement('div')
+    divCardSummery.classList.add('cardSummaryEp')
+    episode.summary ? divCardSummery.innerText =`${episode.summary.replace(/<\/?[^>]+(>|$)/g, "")}` : divCardSummery.innerText= "No Summary" 
+
+    let runtime = document.createElement('div')
+    runtime.classList.add('runtime')
+    runtime.innerText = "runtime: " + episode.runtime + " min"
+        
+    divCard.appendChild(div)
+    divCard.appendChild(imgCard)
+    divCard.appendChild(divCardSummery)
+    divCard.appendChild(runtime)
+
+    rootElem.appendChild(divCard)
+
+    pCount.innerText =`${arr.length}/${arr.length} episodes`
+  }
+
+  async function loadEpisodes(id) {
+    let url = `https://api.tvmaze.com/shows/${id}/episodes`;
+    let obj = await (await fetch(url)).json();
+    episodes = obj
+    
+    episodes.forEach(addEpisodes)
+    
+    pCount.innerText =`${episodes.length}/${episodes.length} episodes`
+  }
```

Comment:
using fetch in this way is great.
this code is perfect, use of foreach loop without element.